packages/components/src/Autocomplete/Autocomplete.tsx

```diff
@@ -16,11 +16,31 @@ interface AutocompleteProps {
    */
   readonly value: Option | undefined;
 
+  /**
+   * Hint text that goes above the value once the form is filled out.
+   */
+  readonly menuSize?: "medium";
+
   /**
    * Hint text that goes above the value once the form is filled out.
    */
   readonly placeholder: string;
 
+  /**
+   * Turn on separators between result elements.
+   */
+  readonly separators?: boolean;
```

Comment:
What are these?

Reply:
![image](https://user-images.githubusercontent.com/49168885/88966722-01653e00-d26a-11ea-877d-1508fa540026.png)
They're the borders between rows

Reply:
What semantic difference is there between autocompletes with separators and ones without? What decides if this should be on or not?

Reply:
We'll have separators show up implicitly if there are details or description.